quilt/tools/store.py

```diff
@@ -68,13 +68,15 @@ def get_package(self, user, package):
         for package_dir in pkg_dirs:
             path = os.path.join(package_dir, user, package + self.PACKAGE_FILE_EXT)
             if os.path.exists(path):
-                return Package(user=user,
-                               package=package,
-                               path=path,
-                               pkg_dir=package_dir)
+                pkg = Package(user=user,
+                              package=package,
+                              path=path,
+                              pkg_dir=package_dir)
+                pkg.load_contents()
+                return pkg
         return None
 
-    def create_package(self, user, package, pkgformat=PackageFormat.default):
+    def create_package(self, user, package, contents):
```

Comment:
Let's have an option to create a package without contents so users can create packages interactively.